src/Model/PrestaShop.php

```diff
@@ -4,15 +4,21 @@
 
 namespace App\Model;
 
+use InvalidArgumentException;
 use JsonSerializable;
 
 class PrestaShop implements JsonSerializable
 {
-    public const CHANNEL_STABLE = 'stable';
-    public const CHANNEL_RC = 'rc';
-    public const CHANNEL_BETA = 'beta';
+    public const string CHANNEL_STABLE = 'stable';
+    public const string CHANNEL_RC = 'rc';
+    public const string CHANNEL_BETA = 'beta';
+
+    public const string FLAVOR_CLASSIC = 'CLASSIC';
+    public const string FLAVOR_OPEN_SOURCE = 'OPEN_SOURCE';
```

Comment:
```suggestion
    public const string FLAVOR_CLASSIC = 'classic';
    public const string FLAVOR_OPEN_SOURCE = 'open_source';
```
Ok for the capitalized const, but values are usually in lower case, just like the values for stability. And since the value will also be part of the JSON format returned by the API I think lower case looks nicer and is more consistent with the other values